src/3d.js

```diff
@@ -33,6 +33,74 @@ The hackOptions below have relatively thorough documentation,
+please) of any length specifying the room stack.
+(rooms in the same stack will be rendered on top of each others, in layers)
+and '12' is the hight at which you want to display a
+specific room. you can use negative numbers and floating points too like
+'#fjk-3.5' (the size of one bitsy tile is 1)
+
+* use wedges to add variety to your scene with stuff like slopes
+and diagonal walls. add the tag to the drawing's name if you want it to
+be displayed in the ⊿ wedge-form ⊿
+for example:
+'#wdg2s' stands for ceiling wedge facing south (with its slanted side)
+'#wdg1nw' for wall wedge facing north-west (with its slanted side)
+0/1/2 stands for the floor/wall/ceiling wedge.
+then to specify direction n/e/s/w for a floor/ceiling wedge,
+and se/sw/nw/ne for a wall wedge
+make sure to read about the next feature that allows for convinient
+use of wedges from the bitsy editor!!!
+
+* repalce drawings with other drawings before they are rendered in 3d
+by appending their names with the tag in this format:
+'#draw_TIL_a'
+where TIL/SPR/ITM is followed by the drawing's id (exactly how you can see it
+in bitsy game data panel. id is a combination of letters and numbers that follows
+'TIL' 'SPR' or 'ITM' like avatar is 'SPR A' in bitsy data and 'A' is the id)
+'#drawTILa' also works
+this is very useful for making gizmo-like drawings that can look different in
+bitsy editor and when rendered in 3d.
+12 variations of wedges can be confusing to work with in bitsy editor, 
+if you can't tell how exactly each wedge is facing based on its drawing.
+with this tag you can have useful descriptive drawings for wedges rotated
+in different directions and then specify drawings you want to use as
+actual textures when rendering your wedges in 3d
+here is a temaplate game data chunk with 12 wedge gizmos that are going to use
+'TIL a' (as it is called in bitsy data) as their texture in 3d instead of their original drawings
+https://gist.github.com/aloelazoe/f1a3b998362a7160cbd74ac31135296a
+just paste it into you game data to use in your bitsy game (add, don't replace existing data)
+or open it in the text editor first and search-replace '#drawTILa'
+to set another drawing you would like to use as a texture on wedges
+if you have a big project with lots of drawings make sure to check that wedges ids
+don't overlap with the ids of the drawings you already have
+
+* move, rotate and scale by adding tags to the drawing's name
+useful for making more complex shapes and more organic silhouettes by shifting
+models a bit off the grid, and configuring plane-type meshes to face a specific direction
+#m(x,y,z) for moving, #r(x,y,z) for rotation (in degrees), #s(x,y,z) for scaling.
+#m(1,0,0.5') and '#m(1,,.5)' are both examples of valid input
+omiting the number is the same as writing 0 for rotation and movement and 1 for scaling
+i.e. it won't change anything on the given axis
+
+* add '#invisible' to the drawing name and it won't be added to the scene,
```

Comment:
i felt like it's not nice to innundate the scene with meshes that have transparent textures (and in cases when you have walkable above ground layers you need quite a lot of them)... as far as i understand they still get rendered and transparency is expensive? or is it not such big of a deal?

Reply:
ah ok, i don't think the performance is a big deal here b/c all the meshes are instanced, but if you noticed a difference, maybe a better way to handle this would be to add an `invisible` entry to the template which returned an empty mesh, rather than relying on the transparency? it would still be slightly less performant than not rendering anything, but would remove the need to check if there actually is a mesh in other parts of the hack